chaoslib/litmus/cpu_hog/pod_cpu_hog.yml

```diff
@@ -0,0 +1,144 @@
+---
+- block: 
+
+    - block: 
+
+      - name: Select the app pod
+        shell: >
+          kubectl get pod -l {{ a_label }} -n {{ a_ns }}
+          -o=custom-columns=:metadata.name --no-headers
+          | shuf | head -1 
+        args:
+          executable: /bin/bash
+        register: app_pod_name
+
+      - name: Record app pod name
+        set_fact:
+          app_pod: "{{ app_pod_name.stdout }}"
+
+      when: "app_pod is not defined"
+
+    # here app_ns is the namespace of pod on which we are performing cpu chaos 
+    # in genric experiments app_ns is same as a_ns
+    # in openebs experiments app_ns is the namespace where openebs is installed i.e, openebs
+    - name: Identify the application node
+      shell: >
+        kubectl get pod {{ app_pod }} -n {{ app_ns }}
```

Comment:
Can you explain why we are using `a_ns` (line no 8) and `app_ns` (line no - 26)

Reply:
fixed!

Reply:
cc: @shubhamchaudhary - hope this is in line with your latest pattern/changes

Reply:
a_ns is the same as app_ns for this experiment.

Reply:
but if we want to use this lib for openebs experiment then a_ns and app_ns may vary

Reply:
in which case we can provide app_ns: "openebs" while doing the include_tasks right?